client/src/pages/Contest/Contest.tsx

```diff
@@ -134,27 +153,28 @@ export default function Contest(): JSX.Element {
           </Grid>
         </Grid>
         <Grid className={classes.grid} container>
-          <Grid item xs={10}>
+          <Grid item xs={12} sm={10}>
             <Typography className={classes.contestTitle}>
-              {contestSubmissions.title ? contestSubmissions.title : 'Lion tattoo concept in minimal style'}{' '}
+              {contest ? contest.title : 'Lion tattoo concept in minimal style'}{' '}
```

Comment:
Can we please remove the hardcoded data in this component?

Reply:
Can do.